dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -104,25 +120,38 @@ private static String generate18(String projectName, double[] percentages, doubl
     /*
      * a private method that generates Figure 19
      */
-    private static String generate19(String projectName, double[] orig_values, double[] time_values, double orig_value,
-            List<GeometricMeanData> fig19GeoData) {
+    private static String generate19(String projectName, double[] orig_values, double[] time_values,
+            double orig_time_value, List<GeometricMeanData> fig19GeoData, boolean[] nonZeroNumOfDTS,
+            double orig_apfd_value) {
         String result = projectName;
         double calc1 = 0;
         double calc2 = 0;
         double calc3 = 0;
```

Comment:
It seems you didn't address Professor Ernst's comment. Please change calc1, calc2 and calc3 to more descriptive names. Ex. calc1 can be enhancedParaSpeedup